src/Streamly/Internal/FileSystem/Handle.hs

```diff
@@ -384,7 +381,7 @@ putStrings encode = putChunks . S.mapM (IA.fromStream . encode . S.fromList)
 putLines :: MonadAsync m
     => (SerialT m Char -> SerialT m Word8) -> SerialT m String -> m ()
 putLines encode = putChunks . S.mapM
-    (\xs -> IA.fromStream $ encode (S.fromList (xs ++ "\n")))
+    (\xs -> IA.fromStream $ encode (S.fromList (xs <> "\n")))
```

Comment:
7.10.3 and 8.0.2 builds are failing due to this. Instead of adding conditional imports to fix this, we can instead suppress this in hlint config.

Reply:
Will fix this. I am trying to work my way through cleaning up Array.hs. Will send both together along with rebasing this.